authors/apps/authentication/tests/test_authentication.py

```diff
@@ -0,0 +1,33 @@
+import json
+
+from django.contrib.auth.models import User
```

Comment:
@Emmanuelaaron I notice User model imported but not used; I suggest we remove the unused imports.

Reply:
Thank you @alexxsanya for taking note of that. I have resolved it.